builtin/providers/aws/resource_aws_opsworks_permission.go

```diff
@@ -0,0 +1,152 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	//"strings"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/opsworks"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsOpsworksPermission() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsOpsworksPermissionCreate,
+		Update: resourceAwsOpsworksPermissionCreate,
+		Delete: resourceAwsOpsworksPermissionDelete,
+		Read:   resourceAwsOpsworksPermissionRead,
+
+		Schema: map[string]*schema.Schema{
+			"id": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"allow_ssh": &schema.Schema{
+				Type:     schema.TypeBool,
+				Computed: true,
+				Optional: true,
+			},
+			"allow_sudo": &schema.Schema{
+				Type:     schema.TypeBool,
+				Computed: true,
+				Optional: true,
+			},
+			"user_arn": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			// one of deny, show, deploy, manage, iam_only
+			"level": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+				Optional: true,
+				ValidateFunc: func(v interface{}, k string) (ws []string, errors []error) {
+					value := v.(string)
+
+					expected := [5]string{"deny", "show", "deploy", "manage", "iam_only"}
+
+					found := false
+					for _, b := range expected {
+						if b == value {
+							found = true
+						}
+					}
+					if !found {
+						errors = append(errors, fmt.Errorf(
+							"%q has to be one of [deny, show, deploy, manage, iam_only]", k))
+					}
+					return
+				},
+			},
+			"stack_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+				Optional: true,
+			},
+		},
+	}
+}
+
+func resourceAwsOpsworksPermissionDelete(d *schema.ResourceData, meta interface{}) error {
```

Comment:
Yes. There is no delete. Also create is not really there, as permissions objects are created on creation of the stack. These are crated with default values an can than be changed.